deploy/helm/values.dev.yaml

```diff
@@ -18,6 +18,8 @@ worker:
   image: acanthamoeba/sds-worker
   queueMode: azure
   queueEndpoint: https://<storageaccount>.queue.core.windows.net/runs
+  resourceGroup: resourceGroup
```

Comment:
This is really the resource group for the entire SDS deployment, not specific to the worker. Can we bump this up a level?